examples/music-bot/src/music/track.ts

```diff
@@ -48,10 +48,10 @@ export class Track implements TrackData {
 			const process = ytdl(
 				this.url,
 				{
-					o: '-',
-					q: '',
-					f: 'bestaudio[ext=webm+acodec=opus+asr=48000]/bestaudio',
-					r: '100K',
+					output: '-',
+					quiet: true,
+					format: 'bestaudio[ext=webm+acodec=opus+asr=48000]/bestaudio',
+					limitRate: '100K',
```

Comment:
Types for the flags were added to `youtube-dl-exec` in [this commit](https://github.com/microlinkhq/youtube-dl-exec/commit/5800d2331034d731ea9387d04efc292476377a18#diff-619cc2239a7eb4ef05041767e6ab0e4fb069e98ee9d2c83c2d8e1865c475e141R250)